From 7d9d35a05a496e0bfbe569c7bedcc0b4306c1b62 Mon Sep 17 00:00:00 2001
From: Hyun Kwon <hyunk@xilinx.com>
Date: Fri, 18 Oct 2013 18:08:43 -0700
Subject: [PATCH 394/827] drm: xilinx: crtc: Remove unneccessary labels

This patch comes from:
  https://github.com/Xilinx/linux-xlnx.git

Remove unnecessary labels.

Signed-off-by: Hyun Kwon <hyunk@xilinx.com>
Acked-by: Michal Simek <michal.simek@xilinx.com>
(cherry picked from commit dc524e1b051b35b79916d283fb44af23c660efbb)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/gpu/drm/xilinx/xilinx_drm_crtc.c | 9 ++++-----
 1 file changed, 4 insertions(+), 5 deletions(-)

diff --git a/drivers/gpu/drm/xilinx/xilinx_drm_crtc.c b/drivers/gpu/drm/xilinx/xilinx_drm_crtc.c
index 56cada9..acb28de 100644
--- a/drivers/gpu/drm/xilinx/xilinx_drm_crtc.c
+++ b/drivers/gpu/drm/xilinx/xilinx_drm_crtc.c
@@ -433,14 +433,14 @@ struct drm_crtc *xilinx_drm_crtc_create(struct drm_device *drm)
 	if (!crtc->si570) {
 		DRM_DEBUG_KMS("failed to get si570 clock\n");
 		ret = -EPROBE_DEFER;
-		goto err_si570;
+		goto err_out;
 	}
 
 	sub_node = of_parse_phandle(drm->dev->of_node, "vtc", 0);
 	if (!sub_node) {
 		DRM_ERROR("failed to get a video timing controller node\n");
 		ret = -ENODEV;
-		goto err_vtc;
+		goto err_out;
 	}
 
 	crtc->vtc = xilinx_vtc_probe(drm->dev, sub_node);
@@ -448,7 +448,7 @@ struct drm_crtc *xilinx_drm_crtc_create(struct drm_device *drm)
 	if (IS_ERR(crtc->vtc)) {
 		DRM_ERROR("failed to probe video timing controller\n");
 		ret = PTR_ERR(crtc->vtc);
-		goto err_vtc;
+		goto err_out;
 	}
 
 	crtc->dpms = DRM_MODE_DPMS_OFF;
@@ -465,8 +465,7 @@ struct drm_crtc *xilinx_drm_crtc_create(struct drm_device *drm)
 
 err_init:
 	xilinx_vtc_remove(crtc->vtc);
-err_si570:
-err_vtc:
+err_out:
 	xilinx_drm_plane_destroy_planes(crtc->plane_manager);
 	xilinx_drm_plane_destroy_private(crtc->plane_manager, crtc->priv_plane);
 err_plane:
-- 
2.9.3

